docs/gravitino-server-config.md

```diff
@@ -37,6 +37,7 @@ The `gravitino.conf` file lists the configuration items in the following table.
 | `gravitino.server.shutdown.timeout`                  | Time in milliseconds to gracefully shut down of the Gravitino webserver.                                                                                                              | `3000`                                                                       | No       | 0.2.0            |
 | `gravitino.server.webserver.customFilters`           | Comma-separated list of filter class names to apply to the API.                                                                                                                       | (none)                                                                       | No       | 0.4.0            |
 | `gravitino.server.rest.extensionPackages`            | Comma-separated list of REST API packages to expand                                                                                                                                   | (none)                                                                       | No       | 0.6.0-incubating |
+| `gravitino.server.visibleConfigs`                    | List of configs that are visible in the config servlet                                                                                                                                | (none)                                                                       | No       | 0.9.0-incubating |
```

Comment:
:) This is a pretty honest name. It can be parsed as "... I have many configs, but I can only show you some ...".

Reply:
OK, I can modify the comment.